.github/workflows/macOS.yml

```diff
@@ -53,3 +56,26 @@ jobs:
           ls -la /usr/local/bin | grep python
           df -h
           brew list
+      # New steps for handling releases on version tags below
+      - name: Check if version tag
+        if: github.ref_type == 'tag'
+        id: check_tag
+        uses: actions-ecosystem/action-regex-match@v2
+        with:
+          text: ${{ github.ref_name }}
+          regex: '^[0-9]+\.[0-9]+\.[0-9]+$'
+          flags: g  # Global flag, but optional if just checking match
+      - name: Generate Github Summary - version tag detected
+        if: steps.check_tag.outputs.match != ''
+        run: |
+          echo "RELEASE DETECTED!" >> $GITHUB_STEP_SUMMARY
+      - name: Generate Github Summary - no version tag
+        if: steps.check_tag.outputs.match == ''
+        run: |
+          echo "Not a Release!" >> $GITHUB_STEP_SUMMARY
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        if: steps.check_tag.outputs.match != ''
+        with:
+          files: |
+              pcp-**/build/mac/pcp-*.dmg
```

Comment:
Is that pattern matching supposed to be two stars up front or would just one do? (like in the second part)

Reply:
This is Javascript 'glob' library which will match multiple segments (e.g. directories, but also by habit I use it for covering off dotted numbers).  I think either way works.